lib/Signal_spec.lua

```diff
@@ -9,52 +9,41 @@ describe("Signal", function()
 
 	it("should handle connections", function()
 		local signal = Signal.new()
-		local callCount = 0
+		local spy = spy.new(function() end)
 
-		local connection = signal:Connect(function()
-			callCount = callCount + 1
-		end)
+		local connection = signal:Connect(spy)
 
 		assert.not_nil(connection)
-		assert.equal(callCount, 0)
 
 		signal:Fire()
-		assert.equal(callCount, 1)
+		assert.spy(spy).was_called()
```

Comment:
What's the default behavior for `was_called`? To me, this reads "this function was called *at least once*", but the behavior we want is "this function was called *exactly once*"

Does this need to change to `was_called(1)`?

Reply:
https://github.com/Olivine-Labs/luassert/blob/73b118e86ca40f1db6dc92b300a68d657c36b39b/src/spy.lua#L51

Yes, it's called at least once. This needs to be changed.

Reply:
Fixed now!